src/main/CustomException/UserNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package CustomException;
+
+public class UserNotFoundException extends Exception {
```

Comment:
Just wondering why we are making a custom exception here, when we could use a UserMessage? or are you trying to do the thing where you throw

Reply:
I’ll have to take another look at why I made this, I don’t think it’s being used anywhere (made it a couple weeks ago lol)